zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServerBean.java

```diff
@@ -40,6 +40,12 @@ public ZooKeeperServerBean(ZooKeeperServer zks) {
         name = "StandaloneServer_port" + zks.getClientPort();
     }
 
```

Comment:
-  `name = "StandaloneServer";` is OK, don't need the port? `StandaloneServer_port2181`? `StandaloneServer_port:2181`, `StandaloneServer_port_2181`:)
-  +1 for the moving of `ObserverBean`
- `ReadOnlyBean`  has been forgotten?